core/src/main/java/edu/wpi/grip/core/StartStoppable.java

```diff
@@ -0,0 +1,52 @@
+package edu.wpi.grip.core;
+
+
+import com.google.common.eventbus.EventBus;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * An Object that can be stopped and started multiple times.
+ */
+public interface StartStoppable {
```

Comment:
These methods should just be in `CameraSource`, since that's the only place they're used.

There might be some overlap if we add networktables sources and we want to be able to start and stop them, but we'd have to write a separate view anyways, so there's wouldn't really be any code sharing